apps/docs/src/app/platform/component-docs/platform-forms/platform-textarea/platform-textarea-examples/platform-textarea-i18n-example.component.html

```diff
@@ -3,37 +3,33 @@
         <fdp-form-group #fg2 [multiLayout]="true" [formGroup]="form">
             <fdp-form-field
                 #ff2
-                [id]="'i18nTextarea'"
+                id="i18nTextarea"
                 i18n-label="@@platformI18nTextareaLabel"
                 label="Detailed description"
                 i18n-placeholder="@@platformI18nTextareaPlaceholder"
                 placeholder="Start entering detailed description"
-                hint="This is tooltip help"
                 zone="zLeft"
                 rank="10"
-                hintPlacement="'left'"
                 [required]="true"
                 [validators]="textareaValidator"
             >
                 <fdp-textarea
                     [formControl]="ff2.formControl"
-                    [name]="'i18nTextarea'"
+                    name="i18nTextarea"
                     [growingMaxLines]="3"
                     [growing]="true"
                     [maxLength]="10"
                     [cols]="10"
-                    [stateType]="'error'"
+                    stateType="error"
                     [showExceededText]="true"
-                    [height]="'80px'"
-                    [wrapType]="'hard'"
+                    height="80px"
+                    wrapType="hard"
                 >
                 </fdp-textarea>
             </fdp-form-field>
 
             <ng-template #i18n let-errros>
-                <span i18n="@@platformI18nRequiredValidation" *ngIf="errros.required">
-                    Value is required
-                </span>
+                <span i18n="@@platformI18nRequiredValidation" *ngIf="errros.required"> Value is required </span>
```

Comment:
`errros` typo

Reply:
this is there in quite a few other places 😅  will fix it here for now